pkg/authn/keychain_test.go

```diff
@@ -107,6 +108,7 @@ func TestPodmanConfig(t *testing.T) {
 	fresh++
 	p := filepath.Join(tmpdir, fmt.Sprintf("%d", fresh))
 	os.Setenv("XDG_RUNTIME_DIR", p)
+	defer os.Unsetenv("XDG_RUNTIME_DIR")
```

Comment:
I think we can use [`t.Setenv`](https://pkg.go.dev/testing#T.Setenv) here instead of `os.Setenv` and defer-unsetting.

Reply:
Nice! Updated.